src/shared/ui/alert/Alert.stories.tsx

```
@@ -0,0 +1,146 @@
+
+export const Error: Story = {
+  name: 'ErrorState/One Button',
+  args: {
+    state: 'error',
+    // 컴포넌트 내부에서 hasTwoBtn은 false로 처리됨
+    title: '문제가 발생했어요',
+    infoText: '네트워크 상태를 확인한 뒤 다시 시도해 주세요.',
+    rightBtnText: '확인',
+    textButtonVariant: 'primary',
+    onRightBtnClick: () => {
+      alert('Error state right button clicked');
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)